packages/db/src/runtime/index.ts

```diff
@@ -8,8 +8,14 @@ import {
 	sqliteTable,
 	text,
 } from 'drizzle-orm/sqlite-core';
-import { type DBColumn, type DBTable } from '../core/types.js';
-import { type SerializedSQL, isSerializedSQL } from './types.js';
+import { tableSchema } from '../core/schemas.js';
```

Comment:
Ah ha, it looks like `tableSchema` would pull a core module into the runtime. We have a lint check against this. Perhaps we can move `asDrizzleTable` from `runtime/` to `core/` and still expose as a package export?

Reply:
Took a quick look, but not sure if that would work? `asDrizzleTable` is used in the Vite plugin:

https://github.com/withastro/astro/blob/cc83367c4fd48ec8bf493d620ef5615780388420/packages/db/src/core/integration/vite-plugin-db.ts#L122

Reply:
OK, tried a different approach, which seems to work. I think the `asDrizzleTable()` we expose to users needs to be different from the one we use internally, so in the latest commit I wrapped the internal version with the tweaks we made for use _only_ by integration authors.